RetailSuiteWarehousing/Integration/01_ConfigureWarehouseGroups.md

```diff
@@ -0,0 +1,186 @@
+
+
+
+## Create a warehouse group
+
+Create a warehouse group to which you can assign any number of specific warehouses, for example a sales warehouse, a returns warehouse, or a storage area for blocked stock.
+
+### Add a warehouse group
+
+Add a new warehouse group to the warehouse group list and define its name and identification number.
+
+#### Prerequisites
+
+The following plugins must be installed:   
+- *Warehouse groups*  
+- *Calculation of the range of storage and ordering by range of storage*  
+- *Warehouse planning*  
+
+#### Procedure
+
+*Warehousing > Settings > Tab WAREHOUSE > Tab Warehouse master data*
+
+![Warehouse master data](../../Assets/Screenshots/RetailSuiteWarehousing/Settings/Warehouse/WarehouseMasterData/WarehouseMasterData.png "[Warehouse master data]")
+
+1. Click the [NEW GROUP] button in the *Warehouse master data* tab.  
+    The *Basic data* sub-tab is displayed.
+
+    ![Warehouse group basic data](../../Assets/Screenshots/RetailSuiteWarehousing/Settings/Warehouse/WarehouseMasterData/WarehouseGroupBasicData.png "[Warehouse group basic data]")
+
+2. Enter a unique number for the warehouse group in the *Warehouse group* field. This is the warehouse group ID.
+
+3. Enter a description for the warehouse group in the *Designation* field. This description will be displayed later on in the left side bar in the *Warehouse master data* tab.
+
+4. Click the *Group type* drop-down list and select the appropriate option. The following options are available:
+    - **Own warehouse group**  
+        Select this option if the warehouse group is managed by yourself.
+    - **Externally controlled warehouse group**  
+        Select this option if the warehouse group is managed by a third party. 
+
+    Depending on the selected option, the fields displayed differ. 
+
+5. For the next steps to create a warehouse group, follow the appropriate procedure:
+
+    - [Create an own warehouse group](#create-an-own-warehouse-group)
+    - [Create an externally controlled warehouse group](#create-an-externally-controlled-warehouse-group)
+
+
+### Create an own warehouse group
+
+Create an own warehouse group, that is, a warehouse group managed by yourself. 
+
+#### Prerequisites
+
+- A warehouse group has been added, see [Add a warehouse group](#add-a-warehouse-group).
+- The **Own warehouse group** option has been selected in the *Group type* field.
+
+#### Procedure
+
+*Warehousing > Settings > Tab WAREHOUSE > Tab Warehouse master data > Select Own warehouse group*
+
+![Own warehouse group basic data](../../Assets/Screenshots/RetailSuiteWarehousing/Settings/Warehouse/WarehouseMasterData/WarehouseGroupBasicData.png "[Own warehouse group basic data]")
+
+1. If necessary, click the *Fulfill orders from other warehouse groups* drop-down list and select a warehouse group. All available warehouse groups are displayed in the list. The **No warehouse group** option includes all warehouses that have not been assigned to any warehouse group.
```

Comment:
Ws für Konsequenzen hat dieses Feld? 
The no warehouse group option... Macht das Sinn?